src/main/java/website/frontrow/artificial/intelligence/PathCalculation.java

```diff
@@ -0,0 +1,120 @@
+package website.frontrow.artificial.intelligence;
+
+import java.util.ArrayList;
+
+import website.frontrow.board.Mover;
+import website.frontrow.board.Enemy;
+import website.frontrow.level.Level;
+import website.frontrow.util.Point;
+
+/**
+ * This class can calculate the best path for an AI to take 
+ * given some info. //TODO: fix comment.
+ * 
+ * This method has no use for a constructor because it always returns a path.
+ */
+@SuppressWarnings("checkstyle:hideutilityclassconstructor")
+
+public class PathCalculation 
```

Comment:
Since all methods in this class are abstract, please consider making it abstract.